docs/blog/2024-07-12-0.10-release.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Releasing 0.10
+description: TBD
+slug: littlehorse-0.10-release
+authors:
+- lh_council
+tags: [littlehorse, release]
+image: https://avatars.githubusercontent.com/u/140006313?s=400&u=7bf4c91d92dfe590ac71bb6b4821e1a81aa5b712&v=4
+hide_table_of_contents: false
+---
+
+The `0.10` release brings with it significant performance and reliability improvements. <!-- truncate -->
+
+## New Features
+
+### `lhctl` Binaries and Release Notes
+
+The `0.10.0` release comes with a new [Release Page](https://github.com/littlehorse-enterprises/littlehorse/releases), including artifacts with `lhctl` built for ARM, Intel, and Windows.
```

Comment:
How about "Including releases of `lhctl` built for ARM, Intel, and Windows."

Artifacts is a heavy word, "releases" is more easily understood.